compiler/rustc_codegen_cranelift/src/base.rs

```diff
@@ -26,10 +26,10 @@ pub(crate) fn codegen_fn<'tcx>(
     let mir = tcx.instance_mir(instance.def);
     let _mir_guard = crate::PrintOnPanic(|| {
         let mut buf = Vec::new();
-        with_no_trimmed_paths!({
```

Comment:
I think this should stay the same. The content of the block is a statement, not an expression. Same for the other changed use of `with_no_trimmed_paths` in cg_clif.